content/docs/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -216,70 +203,124 @@ Registry servers configured in HTTP mode require the `--insecure` CLI flag when
+>>>>>>> 5f11c0ce (docs: resolve conflict)
+
+3. Click on the **Add New Pack Registry**. Palette will open a pop-up window asking for the fields to configure a pack registry server, as highlighted in the screenshot below. 
+
+<<<<<<< HEAD
+  ![A screenshot highlighting the fields to configure a custom pack registry. ](/registries-and-packs_adding-a-custom-registry-tls_certificate.png)
+=======
+7. Click the **Confirm** button to finish configuring the custom pack registry server.
+>>>>>>> 5f11c0ce (docs: resolve conflict)
+
+
+4. Provide the pack registry server name, endpoint, and user credentials in the pop-up window. Ensure to use an "https://" prefix in the pack registry server endpoint. 
+
+
+5. If you want Palette to establish a secure and encrypted HTTPS connection with your pack registry server, upload the certificate in the **TLS Configuration** section. The certificate file must be in the PEM format and have a complete trust chain. 
+
+  If you used a TLS certificate issued by a CA while configuring the pack registry server, check with your CA to obtain a certificate chain from them in the PEM format.
+  
+  If you used a self-signed certificate, upload the entire certificate trust chain as a *.pem* file. If you do not already have one, use a text editor to create a new *.pem* file. Use the following syntax to create the file. The file content must have the server, intermediate, and root certificates concatenated in the specific order mentioned below. The syntax below separates the different certificates using the `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----` comments. Therefore, do not remove those comments from your *.pem* file.  
+  <br />
+
+  ```
+  -----BEGIN CERTIFICATE-----
+  (Your server certificate)
+  -----END CERTIFICATE-----
+
+  -----BEGIN CERTIFICATE-----
+  (Your intermediate certificate)
+  -----END CERTIFICATE-----
+  
+  -----BEGIN CERTIFICATE-----
+  (Your root certificate)
+  -----END CERTIFICATE-----
+  ```
 
-1. Click on **Tenant Settings** > **Certificates** > **Add A New Certificate**
+  Once you upload the *.pem* certificate file and click the **Validate** button, Palette will perform the TLS verification to affirm the certificate's authenticity before establishing a communication channel. 
 
-   Provide the content of your CA Cert (that will be the content of `tls.crt` if
-   you followed this tutorial.
 
-   After you saved the content of the certificate, it will take a few minutes for your
-   custom packs to appear in the available packs list.
+6. Select the **Insecure Skip TLS Verify** checkbox if you do not want an HTTPS connection between Palette and your pack registry server. Suppose you upload a TLS certificate and also select the **Insecure Skip TLS Verify** checkbox. In that case, the **Insecure Skip TLS Verify** checkbox value will take precedence. 
```

Comment:
```suggestion
6. Select the **Insecure Skip TLS Verify** checkbox if you do not want an HTTPS connection between Palette and your pack registry server. If you upload a TLS certificate and also select the **Insecure Skip TLS Verify** checkbox. The **Insecure Skip TLS Verify** checkbox value will take precedence in that case. 
```
Sudhanshu, I know Grammarly suggested "Suppose" but be careful using this. It's often not a good word choice for sentences that are better off starting with an "if".